docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,324 @@
+.. role:: hidden
+    :class: hidden-section
+
+Gender Classification and Eyes Location Detection: A Two Task Problem
+*********************************************************************
+
+.. note::
+
+    - See the notebook `here <https://github.com/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+    - Run in `Google Colab <https://colab.research.google.com/github/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+
+In this example, we are going to implement a multi-task problem. We try to identify the gender of the people, as well as locating their eyes in the image. Hence, we have two different tasks: classification (to identify the gender) and regression (to find the location of the eyes). We are going to use a single network (A CNN) to perform both tasks, however, we will need to apply different loss functions, each proper to a specific task.
+
+Let's import all the needed packages.
+
+.. code-block:: python
+
+    import math
+    import os
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import gdown
+    import zipfile
+    import cv2
+    from natsort import natsorted
```

Comment:
What packages do we need to install for these two?

Reply:
pip install natsort

Reply:
The text should specify that they are needed since they are not usually installed.

Reply:
This was not done in the last commit.

Reply:
I added a text in the IPYNP as I included the codes to install them. in previous examples, you mentioned removing installation codes ( pip install poutyne for example) from the rest file. 
So in the rest file do you want me to just mention the need for installation of also to include the installation codes?

Reply:
Just for these since they are unusual, add a line in the RST saying you use these libraries and add the lines to install them.